pom.xml

```diff
@@ -334,40 +334,40 @@
       <dependency>
         <groupId>org.scala-lang</groupId>
         <artifactId>scala-library</artifactId>
-        <version>2.10.4</version>
+        <version>${scala.version}</version>
       </dependency>
       <dependency>
         <groupId>org.bdgenomics.adam</groupId>
-        <artifactId>adam-core_2.10</artifactId>
+        <artifactId>adam-core_${scala.version.prefix}</artifactId>
```

Comment:
I'm fairly certain there's a reason why we can't do this, I know because I tried to do it once.  Of course, I don't remember why it didn't work . . .